models/monitoring.ts

```diff
@@ -1,20 +1,19 @@
-import { clientMonitorings } from '#clients/monitoring';
+import { clientMonitoring } from '#clients/monitoring';
 import logErrorInSentry from '#utils/sentry';
 import { administrationsMetaData } from './administrations';
+import { IAPINotRespondingError } from './api-not-responding';
 
 export type IRatio = {
+  ratioNumber: number;
   ratio: string;
```

Comment:
Not sure the `ratio` should have both `ratio` and `ratioNumber`. I'm in favour of removing the `string` property and provide only the `number` one. It's the responsability of the view to choose how to display the data (how many decimals for instance).